Makefile

```diff
@@ -23,7 +23,10 @@ FILES     := $$(find . -name "*.go")
 FAILPOINT_ENABLE  := $$(find $$PWD/ -type d | grep -vE "(\.git|tools)" | xargs tools/bin/failpoint-ctl enable)
 FAILPOINT_DISABLE := $$(find $$PWD/ -type d | grep -vE "(\.git|tools)" | xargs tools/bin/failpoint-ctl disable)
 
-default: fmt pairmesh pairportal pairrelay
+default: fmt pairbenchmark pairmesh pairportal pairrelay
+
+pairbenchmark:
```

Comment:
Prefer to user `pairbench` instead of `pairbenchmark`. e.g: `sysbench`.

Reply:
Done